cicd/base/pipeline/deploy_pipeline.sh

```diff
@@ -142,6 +142,7 @@ aws cloudformation deploy \
         pBucketsStackName=$BUCKETS_STACK_NAME \
         pCleanBranch=$clean_branch \
         pEksClusterName=$EKS_CLUSTER_NAME \
+        pEksClusterId=$EKS_CLUSTER_ID \
```

Comment:
Application parameters dont go in the pipeline. If we put app parameters here then every time a parameter changes the pipeline would have to be relaunched. The pipeline is a framework for how the applicaction is launched but it's separate from any application logic, all that comes in at runtime. These values will be imported from regolith and wont need to be in the application stacks either